libraries/disp3D/engine/model/3dhelpers/geometrymultiplier.h

```diff
@@ -0,0 +1,212 @@
+//=============================================================================================================
```

Comment:
why are there 2 geometrymultiplier.h files? I like the one with all lower letters better. please get rid of geometryMultiplier.h